update_slackmoji/update_slackmojis.py

```diff
@@ -0,0 +1,147 @@
+import io
+import json
+import os
+from pprint import pprint
+import sys
+
+from PIL import Image
+import requests
+import yaml
+
+# From https://stackoverflow.com/questions/25108581/python-yaml-dump-bad-indentation
+class MyDumper(yaml.Dumper):
+
+    def increase_indent(self, flow=False, indentless=False):
+        return super(MyDumper, self).increase_indent(flow, False)
+
+def remove_file(f):
+    try:
+        print 'removing file', f
+        os.remove(f)
+    except OSError:
+        pass
+
+
+def download_file(url, output_file):
+    response = requests.get(url)
+    with open(output_file, 'wb') as f:
+        f.write(response.content)
+    return response
+
+
+def write_yaml_file(data, output_file):
+    with open(output_file, "a") as f:
+        yaml.dump(data, f, Dumper=MyDumper, default_flow_style=False)
+
+
+def create_dirs(dir):
```

Comment:
Ummmm, I'll leave that up to someone else - or future me? seems like it could be a different pull request/issue.